.gitignore

```diff
@@ -0,0 +1,7 @@
+soloproj/*
```

Comment:
Give your folders names that describe what is in them. `soloproj` is not descriptive